src/mngt/ocf_mngt_io_class.c

```diff
@@ -41,7 +41,7 @@ int ocf_mngt_add_partition_to_cache(struct ocf_cache *cache,
 	if (cache->user_parts[part_id].config->flags.valid)
 		return -OCF_ERR_INVAL;
 
-	if (max_size > PARTITION_SIZE_MAX)
+	if (max_size > cache->conf_meta->cachelines)
```

Comment:
consider allowing partition max size > cache size. it is tricky to enforce actual limit (don't know actual cache size before attach) and partition max size > cache size makes sense assuming user can detach cache drive and attach larger one.